From a8d7c371e45d542e2643952d7f5c98b8a2dd23ef Mon Sep 17 00:00:00 2001
From: Deepak Kumar <notwho53@gmail.com>
Date: Mon, 28 Apr 2025 00:16:13 +0530
Subject: [PATCH] drop-test-doc

Signed-off-by: Deepak Kumar <notwho53@gmail.com>
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eef1399..48cf045 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,10 +20,8 @@ include_directories(
   include/
 )
 
-add_subdirectory(doc)
 add_subdirectory(data)
 add_subdirectory(include)
-add_subdirectory(tests)
 
 find_package(CoverageReport)
 enable_coverage_report(
-- 
2.49.0
