From 34b34f5dcad1bc0e46bf911bf0c35316e803ce33 Mon Sep 17 00:00:00 2001
From: Deepak Meena <notwho53@gmail.com>
Date: Sun, 25 May 2025 16:48:56 +0530
Subject: [PATCH] drop doc,test,example,qt-api

Signed-off-by: Deepak Meena <notwho53@gmail.com>
---
 sensorfw.pro | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/sensorfw.pro b/sensorfw.pro
index 3c1084c..010ca12 100644
--- a/sensorfw.pro
+++ b/sensorfw.pro
@@ -21,10 +21,7 @@ SUBDIRS = datatypes \
           filters \
           sensors \
           sensord \
-          qt-api \
-          chains \
-          tests \
-          examples
+          chains
 
 contains(CONFIG,configs) {
     contains(CONFIG,hybris) {
@@ -74,10 +71,8 @@ contains(CONFIG,hybris) {
     PKGCONFIGFILES.path = $$[QT_INSTALL_LIBS]/pkgconfig
     QTCONFIGFILES.files = sensord.prf
 
-    qt-api.depends = datatypes
     sensord.depends = datatypes adaptors sensors chains
 
-    include( doc/doc.pri )
     include( common-install.pri )
     include( common-config.pri )
 
-- 
2.49.0

